linera-views/tests/operations_tests.rs

```diff
@@ -37,10 +37,10 @@ async fn test_readings_vec<OP: KeyValueStoreClient + Sync>(
     let mut batch = Batch::new();
     let mut keys = Vec::new();
     let mut set_keys = HashSet::new();
-    for key_value in &key_value_vec {
-        keys.push(key_value.0.clone());
-        set_keys.insert(key_value.0.clone());
-        batch.put_key_value_bytes(key_value.0.clone(), key_value.1.clone());
+    for (key, value) in &key_value_vec {
+        keys.push(&key[..]);
+        set_keys.insert(&key[..]);
```

Comment:
Can you explain to me the use of the syntax `[..]`? Why is it necessary?
Thanks in advance.